demo_waterfall.py

```diff
@@ -17,23 +17,24 @@
 
 from __future__ import division
 import matplotlib.animation as animation
-from matplotlib.mlab import psd
-import pylab as pyl
+import matplotlib.pyplot as plt
 import numpy as np
 import sys
 from rtlsdr import RtlSdr
+from scipy.signal import welch
+from numba import jit
 
 # A simple waterfall, spectrum plotter
 #
 # Controls:
 #
 # * Scroll mouse-wheel up or down, or press the left or right arrow keys, to
-#   change the center frequency (hold shift for finer control).
+#   change the centre frequency (hold shift for finer control).
```

Comment:
All of these spelling and single/double quote changes seem a bit unnecessary.  I don't really mind one way or another, but it just seems like a lot of noise for a functional PR.

Not really a big deal in this case.  For other types of pull requests though (especially in other projects) I would imagine maintainers not being incredibly thrilled.   (FWIW)